app/code/Ludmila/LSCustomField/etc/adminhtml/menu.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="../../../Backend/etc/menu.xsd">
+    <menu>
+        <!--<add id="Ludmila_LSCustomField::main_menu" title="Simple News"-->
```

Comment:
Don't leave commented code if it is not needed. Or add the comment why it is commented, but better remove